README.md

```diff
@@ -81,10 +80,73 @@ It requires only 8 bytes per edge for bi-directional edge access, so, for exampl
 [Twitter-2010](http://law.di.unimi.it/webdata/twitter-2010/) graph of 1.5 billion edges using 
 12GB of RAM.  We use the Tempest library in production, and it has careful unit tests.
 
-If you have any questions about using Tempest or have feature requests, create a Github issue or 
-send an email to <peter.lofgren@cs.stanford.edu>.
+### Created by
+Tempest was built by a team of Stanford PhDs---[Peter Lofgren](@plofgren) (lead developer), [Ashish Goel](@ashishgoel), [Pranav Dandekar](@dandekar)---who have built state-of-the-art systems and algorithms for large scale network analytics at Stanford, Twitter and Amazon. If you have any questions about using Tempest or have feature requests, create a Github issue or send an email to <peter.lofgren@cs.stanford.edu>.
+
+
+## Using Tempest DB (alpha)
+1. Tempest's dependencies (including postgres and java) are neatly packaged in a Docker container.  To
+use Tempest DB, install docker on your machine, then run
+   
+   `docker run -t -i -v ~/:/mnt/home/ -p 127.0.0.1:10001:10001 teapotco/tempestdb:v0.12.0 bash`
+   (Mounting your home directory using `-v ~/:/mnt/home/` is optional, but could be useful for reading
+   your graph files in docker.)  Inside docker, if you've cloned tempest on the host file system, 
+   symlink `~/tempest` to the location under `~/mnt/home/` where tempest is cloned.  Otherwise
```

Comment:
(This is my mistake, but I'll note it here before I forget:)
`~/mnt/home/` => `/mnt/home/`